metricq_wizard_backend/api/views/explorer.py

```diff
@@ -0,0 +1,182 @@
+            metric
+        ]
+
+        for input in self.parse_combinator_expression(config["expression"]):
+            try:
+                if input not in self.nodes:
+                    self.add_metric(input)
+                    await self.search_backwards(input, x_depth - 2)
+                    self.add_layout(input, x_depth - 2)
+                self.add_edge(input, token)
+            except Exception:
+                if input in self.nodes:
+                    del self.nodes[input]
+
+    async def search_aggregator_backwards(self, metric: str, token: str, x_depth: int):
+        if metric not in self.nodes:
+            self.add_metric(metric)
+            self.add_layout(metric, x_depth - 2)
+            await self.search_backwards(metric, x_depth - 2)
+        self.add_edge(metric, token)
+
+    async def search_forwards(self, metric, x_depth):
+        for token in await self.configurator.fetch_consumers(metric):
+            if token not in self.nodes:
+                self.add_agent(token)
+                self.add_layout(token, x_depth + 1)
+            self.add_edge(metric, token)
+
+            if token.startswith("transformer"):
+                if token.endswith("combinator"):
+                    await self.search_combinator_forwards(metric, token, x_depth)
+                elif token.endswith("aggregator"):
+                    await self.search_aggregator_forwards(metric, token, x_depth)
+
+    async def search_aggregator_forwards(self, metric: str, token: str, x_depth: int):
+        metrics = (await self.configurator.get_configs([token]))[token]["metrics"]
+
+        for aggregated_metric, config in metrics.items():
+            if metric == config["source"]:
+                self.insert_metric(aggregated_metric, token, x_depth + 2)
+                await self.search_forwards(aggregated_metric, x_depth + 2)
+
+    async def search_combinator_forwards(self, metric: str, token: str, x_depth: int):
+        metrics = (await self.configurator.get_configs([token]))[token]["metrics"]
+
+        for combined_metric, config in metrics.items():
+            if metric in self.parse_combinator_expression(config["expression"]):
+                self.insert_metric(combined_metric, token, x_depth + 2)
+                await self.search_forwards(combined_metric, x_depth + 2)
+
+    def parse_combinator_expression(self, expression, inputs=None):
+        if inputs is None:
+            inputs = []
+
+        if not isinstance(expression, dict) and not isinstance(expression, list):
+            inputs.append(expression)
+        else:
+            if isinstance(expression, dict):
+                for value in expression.values():
+                    self.parse_combinator_expression(value, inputs)
+            elif isinstance(expression, list):
+                for item in expression:
+                    self.parse_combinator_expression(item, inputs)
```

Comment:
This will also add the value of `operation` and `cooldown_period` to the `inputs` list. See [combinator documentation](https://github.com/metricq/metricq-combinator#usage)